opencensus/common/utils/__init__.py

```diff
@@ -69,6 +69,13 @@ def check_str_length(str_to_check, limit=MAX_LENGTH):
     return (result, truncated_byte_count)
 
 
+def isoz(ts=None):
```

Comment:
to_iso_string? The name isoz could be confusing.

Reply:
Sure, changed in 77a1227.